docs/database-setup.md

```diff
@@ -1,51 +1,80 @@
-* <https://www.postgresql.org/docs/9.5/static/app-createdb.html>
-* <https://www.postgresql.org/docs/9.5/static/app-createuser.html>
+Modify your .env with following URL:
+
+    pgsql://<username>:<password>@localhost/<database name>?serverVersion=9.6
+
+Enjoy!
+
+## Real Deal
+
+Still here? Awesome. Get Some!
+
+     // Create Users
+     CREATE ROLE youwish_devuser WITH LOGIN PASSWORD 'oooh_heees_trying';
+     CREATE ROLE youwish_testuser WITH LOGIN PASSWORD 'oooh_heees_trying';
+     CREATE ROLE youwish_produser WITH LOGIN PASSWORD 'oooh_heees_trying';
+     CREATE ROLE youwish_prodadmin WITH LOGIN PASSWORD 'oooh_heees_trying';
+     // Give all other user's permissions to DB master/root user
+     GRANT youwish_devuser TO youwish_root;
+     GRANT youwish_testuser TO youwish_root;
+     GRANT youwish_produser TO youwish_root;
+     GRANT youwish_prodadmin TO youwish_root;
+     // Give admin prod role everything prod-readWrite has
+     GRANT youwish_produser TO youwish_prodadmin;
+     // Create Databases
+     CREATE DATABASE youwish_prod;
+     REVOKE ALL ON DATABASE youwish_prod FROM public;
+     CREATE DATABASE youwish_dev;
+     REVOKE ALL ON DATABASE youwish_dev FROM public;
+     CREATE DATABASE youwish_test;
+     REVOKE ALL ON DATABASE youwish_test FROM public;
+     // Prod and ProdAdmin can both connect to Prod, but Owner is ProdAdmin
+     ALTER DATABASE youwish_prod OWNER to youwish_prodadmin;
+     GRANT CONNECT ON DATABASE youwish_prod TO youwish_prodadmin;
+     GRANT CONNECT ON DATABASE youwish_prod TO youwish_produser;  
+     // Dev and Test user===admin
+     ALTER DATABASE youwish_dev OWNER to youwish_devuser;
+     ALTER DATABASE youwish_test OWNER to youwish_testuser;
+     GRANT CONNECT ON DATABASE youwish_dev TO youwish_devuser;
+     GRANT CONNECT ON DATABASE youwish_test TO youwish_testuser;
+     // For each db - create schema and default priveleges
+     \connect youwish_prod
+     CREATE SCHEMA schma AUTHORIZATION youwish_prodadmin;
+     SET search_path = schma;
+     ALTER ROLE youwish_prodadmin IN DATABASE youwish_prod SET search_path = schma;
+     ALTER ROLE youwish_produser IN DATABASE youwish_prod SET search_path = schma;
+     GRANT USAGE ON SCHEMA schma TO youwish_produser;
+     GRANT CREATE ON SCHEMA schma TO youwish_prodadmin;
+     ALTER DEFAULT PRIVILEGES FOR ROLE youwish_prodadmin GRANT INSERT, UPDATE, DELETE, TRUNCATE ON TABLES TO youwish_prod;
+     ALTER DEFAULT PRIVILEGES FOR ROLE youwish_prodadmin GRANT USAGE, SELECT, UPDATE ON SEQUENCES TO youwish_prod;
+     GRANT TEMP ON DATABASE youwish_prod TO youwish_produser;
+     //< REPEAT STEPS ^ABOVE^ FOR test and dev, except combine roles since there is no admin/user distinction
+     
+## Manually Copying One DB to Another 
+This is the same for prod to dev/test or one RDS to another one...
+
+```
+   pg_dump -Fc --no-acl -n public -h RDS_URL_BLA_BLA.amazonaws.com -U userthatcanreadolddb dbname > ./sql.dump
```

Comment:
I like this form - this works anywhere.  If we were sticking with RDS, we could do something with snapshots that is even faster and easier, but that is RDS specific.

Note: This will get slow on prod eventually.  When there is a bunch of data and the DB is live.  That's a bridge to cross later though.